### PR TITLE
inject wsrep paths because mysqld doesn't pass them properly

### PR DESCRIPTION
**Pull request by phunehehe**

### CODE REVIEW

File: sdk.nix [hunk 1]
```diff
@@ -92,28 +92,16 @@ rec {
...
+      atop awscli bash binutils cacert coreutils curl diffutils dnsmasq dstat
+      e2fsprogs elasticsearch erlang file findutils gawk gcc glibcLocales
+      gnugrep gnumake gnupg gnused gnutar go gzip htop iftop imagemagick
+      inetutils inotifyTools iotop iperf iproute iptables jdk jetty92 jq kibana
+      kmod libjpeg libjpeg_turbo libxml2 logrotate lsof memcached
+      netcat-openbsd nfs-utils nodejs ntp openjdk openssh openssl optipng
+      parallel perl php postgresql procps psmisc pv pxz python renameutils
+      riemann runit rxvt_unicode s3cmd shadow sshfsFuse stdenv strace
+      strongswan sysstat systemd tcpdump tmux tree tzdata unzip utillinux vim
+      wget xz which rsync;
```
Comment (proger) [inline sdk.nix]: Y?

Comment (proger) [inline sdk.nix]: there was a line per letter

File: pkgs/mariadb-galera/default.nix [hunk 2]
```diff
@@ -32,13 +32,24 @@ stdenv.mkDerivation rec {
   prePatch = ''
     sed -i -e "s|/usr/bin/libtool|libtool|" cmake/libutils.cmake
   '';
-  postInstall = ''
+
+  postInstall =
+  let
+    inherit (import <microgram/sdk.nix>) sdk;
```
Comment (proger) [inline pkgs/mariadb-galera/default.nix]: this is evil

Comment (proger) [inline pkgs/mariadb-galera/default.nix]: packages use callPackage-style deps: https://github.com/zalora/microgram/blob/wsrep-path/pkgs/mariadb-galera/default.nix#L1

File: sdk.nix (hunk 1 above)
Comment (phunehehe) [inline sdk.nix]: there wasn't, but I also see that would be nice, so I'll make it so

Comment (proger) [inline sdk.nix]: ah i see `h` and `i` slipped a bit

File: sdk.nix [hunk 3]
```diff
@@ -107,13 +108,14 @@ rec {
...
       parallel perl php postgresql procps psmisc pv pxz python
-      renameutils riemann runit rxvt_unicode
+      renameutils riemann runit rxvt_unicode rsync
       s3cmd shadow sshfsFuse stdenv strace strongswan sysstat systemd
       tcpdump tmux tree tzdata
       unzip utillinux
       vim
-      wget
-      xz;
+      wget which
```
Comment (proger) [inline sdk.nix:116]: which is a package?.. o_O

Comment (fmap) [inline sdk.nix:116]: https://savannah.gnu.org/projects/which/!

Comment (phunehehe) [inline sdk.nix:116]: apparently :)

**Pull request merged**